tools/CoqMakefile.in

```diff
@@ -286,13 +286,15 @@ ALLNATIVEFILES = \
 	$(OBJFILES:.o=.cmi) \
 	$(OBJFILES:.o=.cmx) \
 	$(OBJFILES:.o=.cmxs)
-# trick: wildcard filters out non-existing files
-NATIVEFILESTOINSTALL = $(foreach f, $(ALLNATIVEFILES), $(wildcard $f))
+# trick: wildcard filters out non-existing files, so that `install` doesn't show
+# warnings and `clean` doesn't pass to rm a list of files that is too long for
+# the shell.
+NATIVEFILES = $(foreach f, $(ALLNATIVEFILES), $(wildcard $f))
```

Comment:
Why not just do `$(wildcard $(ALLNATIVEFILES))`?

Reply:
Right. This was already there, but should be improved.